src/waitress/adjustments.py

```diff
@@ -392,20 +430,20 @@ def __init__(self, **kw):
                         wanted_sockets.append((family, socktype, proto, sockaddr))
                         hp_pairs.append((sockaddr[0].split("%", 1)[0], sockaddr[1]))
 
-            except Exception:
-                raise ValueError("Invalid host/port specified.")
+            except Exception as exc:
+                raise ValueError("Invalid host/port specified.") from exc
```

Comment:
fwiw I do not believe that the `from exc` is necessary in these cases you have modified - it is an explicit form of what already happens when raising a new exception from within an except block.

Reply:
I setup a new linter (not sure which one) and it suggested I try this.